Tasks/SqlAzureDacpacDeploymentV1/task.json

```diff
@@ -79,13 +79,29 @@
             "helpMarkDown": "Target Azure Resource Manager subscription for deploying SQL files",
             "visibleRule": "ConnectedServiceNameSelector = ConnectedServiceNameARM"
         },
+        {
+            "name": "TargetMethod",
+            "type": "pickList",
+            "label": "Authentication",
+            "required": true,
+            "groupName": "target",
+            "defaultValue": "server",
```

Comment:
Yes, it will be a required input.

Reply:
In Yaml, this will be a extra input user has to give. I presume, we can infer the authentication type during execution based on appropriate subsequent input given in Yaml (like if username/password is specified). For designer UI - I agree we need this input so that it can help in visibility.

Reply:
Can you talk to @SumiranAgg on this. @SumiranAgg  had done similar optimizations of input for Azure App service task.

Reply:
@kmkumaran  

This input is not only used for visibility. E.g. in case of `SQL server authentication`  and `Active directory - Password authentication` all other input fields are same. This input is only deciding factor even in YAML flow.  That's why we can't remove this input in YAML flow.